packages/experimental/Expander/README.md

```diff
@@ -0,0 +1,452 @@
+* 'right': The content is aligned to the right of the Expander.
+* 'stretch': The content will stretch to fit the Expander.
+
+| Type                                       | Required |
+| ------------------------------------------ | -------- |
+| enum('center', 'left', 'right', 'stretch') | No       |
+
+---
+
+### `contentVerticalAlignment`
+
+Sets the vertical alignment of the Expander's content.
+
+* 'bottom': The content is aligned to the bottom of the Expander.
+* 'center': The content is aligned in the center of the Expander.
+* 'stretch': The content will stretch to fit the Expander.
+* 'top': The content is aligned to the top of the Expander.
+
+| Type                                       | Required |
+| ------------------------------------------ | -------- |
+| enum('bottom', 'center', 'stretch', 'top') | No       |
+
+---
+
+### `headerBackground`
+
+Header background color. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `headerForeground`
+
+Header foreground color at rest. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `headerForegroundPointerOver`
+
+Header foreground color on pointer over. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `headerForegroundPressed`
+
+Header foreground color when pressed. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `headerBorderBrush`
+
+Header border color at rest. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `headerBorderPointerOverBrush`
+
+Header border color on pointer over. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `headerBorderPressedBrush`
+
+Header border color when pressed. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `headerDisabledForeground`
+
+Header foreground color when disabled. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `headerDisabledBorderBrush`
+
+Header border color when disabled. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `headerBorderThickness`
+
+Header border thickness.
+
+| Type   | Required |
+| ------ | -------- |
+| number | No       |
+
+---
+
+### `contentBackground`
+
+Content background color. Input value should be hexadecimal string.
+
+| Type   | Required |
+| ------ | -------- |
+| string | No       |
+
+---
+
+### `contentBorderBrush`
+
+Content border color. Input value should be hexadecimal string.
```

Comment:
I thought that you could also pass string names of colors - can you check if that works?